app/mailers/notification_mailer.rb

```diff
@@ -1,166 +1,161 @@
 class NotificationMailer < ApplicationMailer
```

Comment:
I thought the same.. once we have the entire class covered I will try to improve this.